WireAPI/Sources/WireAPI/Private/APIs/BackendInfoAPI/BackendInfoAPIV2.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class BackendInfoAPIV2: BackendInfoAPIV1 {
```

Comment:
How will we decide which `BackendInfoAPI` version we should use? Since the `/api-version` is unversioned doesn't it make to have only one BackendInfoAPI implementation where the `var development: [UInt]` is optional?

Reply:
I get that it serves as an example here so maybe it makes sense to keep initially until we've implemented another versioned API .

Reply:
I will create a ticket to make it an unversioned API once we no longer need it as an example, does that sound ok?